judgments/tests.py

```diff
@@ -318,3 +318,15 @@ def test_min_max():
     assert views.as_integer(2, minimum=1, maximum=3) == 2
     assert views.as_integer(None, minimum=1, default=4) == 4
     assert views.as_integer(None, minimum=1) == 1
+
+
+def test_prep_query():
+    assert utils.remove_unquoted_stop_words("weight of evidence") == "weight evidence"
+    assert (
+        utils.remove_unquoted_stop_words("'weight of evidence'")
+        == "'weight of evidence'"
+    )
+    assert utils.remove_unquoted_stop_words("the") == ""
+    assert utils.remove_unquoted_stop_words("'the'") == "'the'"
+    assert utils.remove_unquoted_stop_words("judge and jury") == "judge jury"
```

Comment:
Is `cats and dogs "kittens and puppies"` a meaningful search term we should check?

Reply:
Good question!!